lib/tensorflow/session.rb

```diff
@@ -17,9 +17,9 @@ class Tensorflow::Session
   # A TensorFlow graph is a description of computations. To compute anything, a graph must be launched in a Session. A Session places the graph ops and provides methods to execute them.
 
   def initialize()
-  	self.status = Tensorflow::TF_NewStatus()
-  	self.ops = Tensorflow::TF_NewSessionOptions()
-  	self.session = Tensorflow::TF_NewSession(self.ops, self.status)
+  	self.status = Tensorflow::Internal::TF_NewStatus()
+  	self.ops = Tensorflow::Internal::TF_NewSessionOptions()
+  	self.session = Tensorflow::Internal::TF_NewSession(self.ops, self.status)
```

Comment:
Use 2 spaces here instead of tabs. So it becomes:

```ruby
def initialize
  self.status = Tensorflow::Internal::TF_NewStatus()
  self.ops = Tensorflow::Internal::TF_NewSessionOptions()
  self.session = Tensorflow::Internal::TF_NewSession(self.ops, self.status)
end
```

Oh yeah, and remove `()` when no arguments in `initialize`